lib/galaxy/webapps/galaxy/config_schema.yml

```
@@ -1004,6 +1004,19 @@ mapping:
         desc: |
           Enable InteractiveTools.
 
+      interactivetools_proxy_host:
+        type: str
+        desc: |
+          Proxy host - assumed to just be hosted on the same hostname and port as
+          Galaxy by default.
```

Would this be correct ?
```suggestion
          Proxy hostname and port. Use this if you run gx-it-proxy outside of Galaxy.
          If not specified defaults to the Galaxy hostname and port.
```
And can you rebuild this config files with these changes ?